jpo-ode-svcs/src/main/java/us/dot/its/jpo/ode/traveler/TimDepositController.java

```diff
@@ -120,7 +120,7 @@ public synchronized ResponseEntity<String> depositTim(String jsonString, Request
       ServiceRequest request;
       try {
          // Convert JSON to POJO
-         odeTID = (OdeTravelerInputData) JsonUtils.fromJson(jsonString, OdeTravelerInputData.class);
+         odeTID = (OdeTravelerInputData) JsonUtils.jacksonFromJson(jsonString, OdeTravelerInputData.class, true);
```

Comment:
Is that RSU additional field from the JSON lost once it is converted into POJO?

Reply:
Yes. Any unrecognized properties will be dropped.

We're giving the endpoint extra properties for an object that it doesn't recognize. Previously when we were using `gson` the unrecognized properties were ignored. When we switched to using `jackson`, unrecognized properties caused an error.

This brings the functionality back to where it should be: ignoring the unrecognized properties.